pkg/management/component_test.go

```diff
@@ -1382,15 +1401,105 @@ func TestGetGroupsOfUser(t *testing.T) {
 	}
 }
 
+func TestSetTrustIDGroups(t *testing.T) {
+	var mockCtrl = gomock.NewController(t)
+	defer mockCtrl.Finish()
+	var mockKeycloakClient = mock.NewKeycloakClient(mockCtrl)
+	var mockEventDBModule = mock.NewEventDBModule(mockCtrl)
+	var mockConfigurationDBModule = mock.NewConfigurationDBModule(mockCtrl)
+	var mockLogger = log.NewNopLogger()
+	var allowedTrustIDGroups = []string{"grp1", "grp2"}
+
+	var managementComponent = NewComponent(mockKeycloakClient, mockEventDBModule, mockConfigurationDBModule, allowedTrustIDGroups, mockLogger)
+
+	var accessToken = "TOKEN=="
+
+	var username = "user"
+	var realmName = "master"
+	var userID = "789-1234-5678"
+
+	// Set groups with success
+	{
```

Comment:
You can use t.Run(...) (something we recently discovered :-) )
```
t.Run("Set groups with success", func(t *testing.T) {
    // Your test
})
```

Reply:
Good to know, fixed!